tests/testLoginManager.php

```diff
@@ -133,4 +173,133 @@ function ( $auth_url, $params ) {
 		);
 		$this->assertEquals( 'https://test-custom.auth0.com/authorize?test=this', $auth_url );
 	}
+
+	/**
+	 * Test that logged-in users are redirected from the wp-login.php page.
+	 */
+	public function testThatLoggedInUserIsRedirectedFromWpLogin() {
+		$this->startRedirectHalting();
```

Comment:
This is what I didn't fully understand from yesterday's response. Couldn't you call this line at the very end of the "run before each test" method? It's cleaner than having to copy-paste it on every test.

Reply:
Cleaner but not needed for every test. Some of these cannot be combined because they throw exceptions at different places.